cdap-cli/src/main/java/co/cask/cdap/cli/util/ArgumentParser.java

```diff
@@ -47,7 +47,7 @@
     if (mapString == null || mapString.isEmpty()) {
       return ImmutableMap.of();
     }
-    return Splitter.on(" ").withKeyValueSeparator("=").split(mapString);
+    return Splitter.on(",").omitEmptyStrings().trimResults().withKeyValueSeparator("=").split(mapString);
```

Comment:
We will have to double-check the examples and documentation. Also, we'll need to decide on the format. I prefer "key1=a key2=b" since it's a bit less to type.